pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -3147,3 +3147,52 @@ def test_require_integers(offset_types):
     cls = offset_types
     with pytest.raises(ValueError):
         cls(n=1.5)
+
+
+def test_comparisons(offset_types):
```

Comment:
I think this needs to be broken up a bit, its very hard to read / determine what is allowed and what is not.

IOW you need 3 tests
Week, Tick, Everything else

Reply:
and the Tick test should be in tests_ticks (and you have one already there)